src/EnergyPlus/DataHeatBalSurface.hh

```diff
@@ -67,20 +67,40 @@ namespace DataHeatBalSurface {
     extern Real64 const DefaultSurfaceTempLimit;        // Highest inside surface temperature allowed in Celsius
     extern std::vector<bool> Zone_has_mixed_HT_models;  // True if any surfaces in zone use CondFD, HAMT, or Kiva
 
-    // DERIVED TYPE DEFINITIONS
-
-    // MODULE VARIABLE DECLARATIONS:
-
-    // SUBROUTINE SPECIFICATIONS FOR MODULE DataHeatBalSurface
     // Integer Variables for the Heat Balance Simulation
     extern Array1D_int SUMH; // From Old Bldctf.inc
 
+    // Surface heat balance limits and convergence parameters
+    extern Real64 MaxSurfaceTempLimit;               // Highest inside surface temperature allowed in Celsius
+    extern Real64 MaxSurfaceTempLimitBeforeFatal;    // 2.5 times MaxSurfaceTempLimit
+    extern Real64 const IterDampConst;               // Damping constant for inside surface temperature iterations
+    extern int const ItersReevalConvCoeff;           // Number of iterations between inside convection coefficient reevaluations
+    extern int MinIterations;                        // Minimum number of iterations for surface heat balance
+    extern int const MaxIterations;                  // Maximum number of iterations allowed for inside surface temps
+    extern Real64 const PoolIsOperatingLimit;        // Limit to determine if swimming pool is operating or not
+    extern int const MinEMPDIterations;              // Minimum number of iterations required for EMPD solution
+    extern int const IterationsForCondFDRelaxChange; // number of iterations for inside temps that triggers a change
```

Comment:
Moved these here from `static` declarations in `CalcHeatBalanceInsideSurf*`.

Reply:
YAY!